sandbox/version-control-manager/src/main/java/com/vikas/method_parser/version_control_manager/details/fetcher/util/RESTCallHandler.java

```diff
@@ -10,6 +14,7 @@
  * Used to make Rest Service calls.
  *
  */
+@Path("RESTCallHandler")
```

Comment:
This shows a lack of understanding of API concepts, remove this. This is not a controller.